plugins/nitf/test/NitfReaderTest.cpp

```diff
@@ -166,10 +166,10 @@ TEST(NitfReaderTest, optionSrs)
     writer.setInput(*nitfReader);
     writer.setOptions(lasOpts);;
 
-    writer.prepare(ctx);
-    PointBufferSet pbSet = writer.execute(ctx);
+    writer.prepare(table);
+    PointViewSet pbSet = writer.execute(table);
 
     EXPECT_EQ(sr, nitfReader->getSpatialReference().getWKT());
     EXPECT_EQ("", writer.getSpatialReference().getWKT());
-    EXPECT_EQ(sr, ctx.spatialRef().getWKT());
+    EXPECT_EQ(sr, table.spatialRef().getWKT());
```

Comment:
`Error	842	error C2039: 'spatialRef' : is not a member of 'std::shared_ptr<_Ty>'`